Gemini.md don't accept @file anymore

it seems that gemini-cli has changed something 


```
[ERROR] [ImportProcessor] Failed to import file: ENOENT: no such file or directory, access '/Users/chmouel/git/pac/trees/second/file'
```

--- a/GEMINI.md
+++ b/GEMINI.md
@@ -1,1 +1,12 @@
-@file .cursor/rules/*
+@.cursor/rules/architecture.mdc
+@.cursor/rules/code-formatting.mdc
+@.cursor/rules/dependencies.mdc
+@.cursor/rules/documentation.mdc
+@.cursor/rules/e2e-testing.mdc
+@.cursor/rules/git-commit-format.mdc
+@.cursor/rules/jira-formatting.mdc
+@.cursor/rules/pre-commit.mdc
+@.cursor/rules/srvkp-jira-bug-template.mdc
+@.cursor/rules/srvkp-jira-template.mdc
+@.cursor/rules/testing-quality.mdc
+@.cursor/rules/useful-commands.mdc

